### PR TITLE
Rename `VBMetadata`

### PR DESCRIPTION
**Pull request by alexander-yevsyukov**
This PR improves naming of classes related to generating validating builders.

### CODE REVIEW

**Comment by alexander-yevsyukov** [comment]
@armiol, I cannot advance the version according to the instructions left by @dmdashenkov in `ext.gradle`. Please give some thought as to making it easier that it's now.

**Comment by alexander-yevsyukov** [comment]
@armiol, PTAL.

File: gradle-plugins/model-compiler/src/main/java/io/spine/tools/compiler/validation/ClassNames.java [hunk 1]
```diff
@@ -50,27 +51,26 @@ private ClassNames() {
      * Returns the {@code ClassName} for the Protobuf field
      * based on the passed {@code FieldDescriptorProto}.
      *
-     * @param fieldDescriptor  the field descriptor of the Protobuf field
-     * @param messageTypeCache the cache of the message types
+     * @param field  the field descriptor of the Protobuf field
```
Comment (armiol) [inline gradle-plugins/model-compiler/src/main/java/io/spine/tools/compiler/validation/ClassNames.java]: This line seems unformatted.

File: gradle-plugins/model-compiler/src/main/java/io/spine/tools/compiler/validation/VBTypeLookup.java [hunk 2]
```diff
@@ -38,94 +40,96 @@
 import static com.google.common.collect.Sets.newHashSet;
 
 /**
- * Assembles the {@code VBMetadata}s from the Protobuf.
+ * Collects type for which generated validating builders.
```
Comment (armiol) [inline gradle-plugins/model-compiler/src/main/java/io/spine/tools/compiler/validation/VBTypeLookup.java]: Probably, `Collects types for which validating builders are generated.`

**Pull request merged**